src/factories/inputs/output-row/FactoryOutputRow.tsx

```diff
@@ -151,8 +151,6 @@ export function FactoryOutputRow(props: IFactoryOutputRowProps) {
         <IconTrash size={16} stroke={1.5} />
       </ActionIcon>
 
-      <OutputDependenciesPeekModal factoryId={factoryId} output={output} />
```

Comment:
Is this reimplemented somewhere else?

Reply:
No, I will add it to the details view and back in this view too